src/lib/dfp/Advert.ts

```diff
@@ -100,6 +100,7 @@ class Advert {
 	hasPrebidSize = false;
 	headerBiddingBidRequest: Promise<unknown> | null = null;
 	lineItemId: number | null = null;
+	testgroup: string | undefined;
```

Comment:
I think this might benefit from a comment to indicate that this is an Ozone testgroup. It isn't obvious that this property is fo

Reply:
I totally agree with you! Done 130c68de8204e39f925964f88b5b66601b58c446